### PR TITLE
Fix typos in SET docs

### PR DESCRIPTION
**Pull request by mickmister**

### CODE REVIEW

**Comment by mickmister** [comment]
@justinegeffen Is this supposed to point to master? Clicking "Edit on GitHub" made it point to the 0.2.1 branch

**Comment by justinegeffen** [comment]
> @justinegeffen Is this supposed to point to master? Clicking "Edit on GitHub" made it point to the 0.2.1 branch

0.2.1 is the master branch in the Handbook. :)

**Review by justinegeffen — APPROVED** [review]

File: operations/research-and-development/engineering/team/README.md [hunk 1]
```diff
@@ -40,23 +40,22 @@ Work for SET comes from various sources:
...
+* Make sure reported bugs are in fact bugs and recommend turning non-bugs into stories or feature requests.
 
 ## Priority of Work
 
 Below is how SET prioritizes what is worked on.
 
 1. Quality issues affecting community.mattermost.com or community-daily.mattermost.com
-   * When a high impact issue affects the community server, SET responds and is responsible to coordinate fixing
-   * The primary goal is to restore stability, whether this is achieved by fixing the issue or reverting code
+   * When a high impact issue affects the community server, SET responds and is responsible to coordinate fixing.
```
Comment (mickmister) [inline operations/research-and-development/engineering/team/README.md:52]:
```suggestion
   * When a really high impact issue affects the community server, SET responds and is responsible to coordinate fixing.
```

**Comment by mickmister** [comment]
@justinegeffen I don't have permissions to merge this, can you please merge?

**Pull request merged**